packages/grafana-ui/src/options/builder/grid.tsx

```diff
@@ -0,0 +1,23 @@
+import { PanelOptionsEditorBuilder } from '@grafana/data';
+import { VizGridLines, OptionsWithGridLines } from '@grafana/schema';
+
+/**
+ * @public
+ */
+export function addGridLinesOptions<T extends OptionsWithGridLines>(builder: PanelOptionsEditorBuilder<T>) {
+  builder.addRadio({
+    path: 'grid',
```

Comment:
i would much prefer if this was `grid.show or `showGrid`. we may want to add more grid options in the future and having a general `grid` prop for visibility feels not future-proof enough.

Reply:
yeah, this sounds reasonable to me